zuul.d/projects.yaml

```diff
@@ -4,5 +4,7 @@
     github-check:
       jobs:
         - dataplane-operator-docs-preview
+        - data-plane-adoption-github-rdo-centos-9-crc-single-node:
```

Comment:
I wonder if it is testing the dataplane-operator from the PR or if it is testing the dataplane that is pinned on openstack-operator

Reply:
afaik the job should be getting a new build from the CI framework.  https://review.rdoproject.org/r/plugins/gitiles/rdo-jobs/+/refs/heads/master/playbooks/data_plane_adoption/setup_crc_use_install_yamls.yaml#26 I suppose @jistr would have more info about that.

Reply:
I don't know the answer to that but maybe @marios or @cescgina could.

One more thing i'd say is we should be adding the extracted adoption job, not the single node one. I think the extracted job is meant to supersede the single node one eventually, as it runs CRC without nested virt.

Reply:
@jistr is right, that in case of adding an adoption job, it should be the extracted crc one and not the single node. I have a PR open in install_yamls to setup the job there: https://github.com/openstack-k8s-operators/install_yamls/pull/608
right now, the job running is not consuming the output of the content-provider, since it's not listed as a dependency

Reply:
Ok. Switching to `data-plane-adoption-github-rdo-centos-9-extracted-crc` .